### DISCUSSION TITLE
请问字节跳动的列表有吗？

### DISCUSSION THREAD

**Discussion by ohbaby30** [v2fly/domain-list-community#941] [General] [answered]
请问字节跳动的列表有吗？

**Comment by rootmelo92118** [comment, marked as answer]
geosite:bytedance

**Comment by Travel2Here** [comment]
有是有，包含关系有点乱（也许我没搞明白，请指点一二）。
目标想要大陆白名单，按目前看最大集合是geosite:cn。递归翻下去，这里面包含geolocation-cn，继续包含bytedance，表里又包含tiktok（显然这个不在大陆可访问范围）。
所以这白名单到底用哪个 or 怎么用才对？（主服务于大陆直连）

**Reply by rootmelo92118** [reply]
建議你可以在geosite:cn 的規則前特別幫tiktok 進行分流。

**Reply by Travel2Here** [reply]
> 建議你可以在geosite:cn 的規則前特別幫tiktok 進行分流。

谢谢，只能说可以解决问题就是有点蠢 哈哈
